RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -105,10 +105,10 @@ boolean auto_haveSITCourse()
 void auto_SITCourse()
 {
 	if (!auto_haveSITCourse()) return;
-	//Best choice seems to be insectologist
-	if (!have_skill($skill[insectologist])){
+	//Get cryptobotanist if under level 8 or switch to insectologist if possible
```

Comment:
Bonus change which is optional. Add usage of `buffMaintain($effect[extra-green]);` at line 1980 of auto_restore. Very similar to usage of Tingling Insides on line 1776

Reply:
Good call